library/core/class.form.php

```diff
@@ -367,8 +367,11 @@ public function categoryDropDown($FieldName = 'CategoryID', $Options = false) {
                     $ForceCleanSelection = false;
                 }
 
+                $Disabled = false;
```

Comment:
This is not correct - you are overriding any previous setting of disabled. Look just 7 lines above this.